nomenclature/cli.py

```diff
@@ -154,3 +154,42 @@ def check_region_aggregation(
         results_df.to_excel(processed_data)
     if differences:
         differences_df.reset_index().to_excel(differences, index=False)
+
+
+@cli.command("export-project")
```

Comment:
I would call it `export-definition`, we've used `project` in the past to mean an entire workflow directory, including model mappings, required data, etc...

Reply:
Ok.